appinfo/info.xml

```diff
@@ -53,6 +53,8 @@ Note: encrypting the contents of group folders is currently not supported.]]></d
 	<settings>
 		<admin>OCA\GroupFolders\Settings\Admin</admin>
 		<admin-section>OCA\GroupFolders\Settings\Section</admin-section>
+		<personal>OCA\GroupFolders\Settings\Admin</personal>
```

Comment:
This is the way I've found now to make groupfolders accessible to user. But, it has the side-effect that groupfolders doesn't appear in the admin settings anymore

Reply:
I think we must create a new setting page and not move it the existing.

What do you think ?

Reply:
Probably. But I'm reluctant because it's probably a lot of work (in react.js which I don't know at all)

Reply:
No, you were right: creating the new setting page didn't involved any react.js writing ;-)